documentation/configuring_bolt.md

```diff
@@ -217,6 +217,7 @@ Bolt uses the following precedence when interpolating configuration settings,
 from highest precedence to lowest:
 
   - Target URI (i.e. ssh://user:password@hostname:port)
+  - [Plan function](plan_functions.md) options (ie. `_run_as`)
```

Comment:
```suggestion
  - [Plan function](plan_functions.md) options (i.e. `_run_as`)
```

Reply:
I think there's an opportunity here to clarify these bullet points a little further, while also removing these latin abbreviations (they're not great for localization + accessibility):
```
Bolt uses the following precedence when interpolating configuration settings,
from highest precedence to lowest:

  - Configuration specifications from the target's URI. For example, `ssh://user:password@hostname:port`.
  - [Plan function](plan_functions.md) options that modify configuration, such as `_run_as`.
  - [Inventory file](inventory_file_v2.md) configuration options.
  - [Command line flags](bolt_command_reference.md) that modify configuration.
  - Options from the project-level configuration file, `bolt-project.yaml`. 
  - Options from the user-level configuration file, `~/.puppetlabs/etc/bolt/bolt-defaults.yaml`.
  - Options from the system-wide configuration file, `/etc/puppetlabs/bolt/bolt-defaults.yaml`.
  - SSH configuration file options. For example, `~/.ssh/config`.
```